panther_docking/config/panther_docking_server.yaml

```diff
@@ -16,23 +16,32 @@
     dock_plugins: ["panther_charging_dock"]
     panther_charging_dock:
       plugin: panther_docking::PantherChargingDock
-      external_detection_timeout: 0.1
+      external_detection_timeout: 0.2
       docking_distance_threshold: 0.12
-      docking_yaw_threshold: 0.15
-      staging_x_offset: -0.5
+      docking_yaw_threshold: 0.2
+      staging_x_offset: -0.8
       filter_coef: 0.1
       use_wibotic_info: <use_wibotic_info_param>
       wibotic_info_timeout: 1.0
 
-    docks: ["main"]
+    docks: ["main", "backup"]
     main:
       type: panther_charging_dock
-      frame: <robot_namespace>/main_wibotic_receiver_requested_pose_link
-      pose: [0.0, 0.0, 0.0] # position of the dock device (not the staging position), the front (X+) of the dock should point away from the robot
+      frame: <robot_namespace>/map
+      dock_frame: main_wibotic_receiver_requested_pose_link
+      pose: [1.0, 1.5, 1.57] # position of the dock on the map. Used also for spawning dock in the simulation.
+      apriltag_id: 0
+
+    backup:
+      type: panther_charging_dock
+      frame: <robot_namespace>/map
+      dock_frame: backup_wibotic_receiver_requested_pose_link
+      pose: [-1.0, 1.5, 1.57]  # position of the dock on the map. Used also for spawning dock in the simulation.
```

Comment:
Also it is not intuitive the array contain x,y,yaw elements. Probably most of the people will interpret this as a x,y,z array.

Reply:
This is defined in the `opennav_docking`. I added the comment that it is a pose and it has [x, y. yaw]